xarray/core/duck_array_ops.py

```diff
@@ -72,9 +72,7 @@ def fail_on_dask_array_input(values, msg=None, func_name=None):
         raise NotImplementedError(msg % func_name)
 
 
-# switch to use dask.array / __array_function__ version when dask supports it:
-# https://github.com/dask/dask/pull/4822
-moveaxis = npcompat.moveaxis
```

Comment:
no, I wouldn't think so: `np.moveaxis` dispatches through `__array_function__`, and we don't define that on `xarray` objects (see #3917)

Reply:
Then I'm not sure whether to just close this pr because it won't change anything or just view this as removing unused code?

Reply:
Very good question! 

I can't find many places in the code that this is called, but here's one: https://github.com/pydata/xarray/blob/v0.18.2/xarray/core/variable.py#L782, so either it works under some conditions, or that is never called...

Reply:
I think what the comment is talking about is that `npcompat.moveaxis` is not necessary once `np.moveaxis(dask_array, ...)` works (which it does since the merge of the linked PR).